CONTRIBUTING.md

```diff
@@ -57,13 +92,41 @@ Make sure to [keep your fork up to date][link_updateupstreamwiki] with the maste
 
 Try to keep the changes focused. We've found that working on a [new branch][link_branches] makes it easier to keep your changes targeted.
 
-When you're creating your pull request, please make sure to review the tedana [style conventions][link_styleguide].
+When you're creating your pull request, please make sure to review the tedana [style conventions](#style-guide).
 
 **4. Submit a [pull request][link_pullrequest].**
 
-A member of the development team will review your changes to confirm that they can be merged into the main codebase.
+A member of the development team will review your changes to confirm that they can be merged into the main code base.
+When opening the pull request, we ask that you follow some [specific conventions](#pull-requests).
+We outline these below.
+
+### Pull Requests
+
+To improve understanding pull requests "at a glance", we encourage the use of several standardized tags.
+When opening a pull request, please use at least one of the following prefixes:
+
+* **[ENH]** for enhancements
+* **[FIX]** for bug fixes
+* **[TST]** for new or updated tests
+* **[DOC]** for new or updated documentation
+* **[STY]** for stylistic changes
+* **[RF]** for refactoring existing code
```

Comment:
There's a paragraph below that mentions WIP, but I think it _would_ be clearer if WIP was included in the bulleted list.

Also, on an unrelated note (but unfortunately linked to this exact line), I would like to request that we change RF to REF. Everything else is three letters, so I think it would look better.

Reply:
I like the idea of adding `[WIP]` to the list and totally fine with `[REF]` instead of `[RF]`.

I also like the combinations! @emdupre, @tsalo - what format do you both currently use? I think I've seen commas between the different categories, right? Eg: `[WIP, DOC, TST]`?

Reply:
FWIW, we don't bother having `[]` decoration and just add `:` at the end in the datalad https://github.com/datalad/datalad/blob/master/CONTRIBUTING.md#how-to-contribute
Saves us an entire character